darwin/torch/dataset.py

```diff
@@ -331,7 +331,9 @@ def get_target(self, index: int) -> Dict[str, Any]:
         annotations = []
         for obj in target["annotations"]:
             sequences = convert_polygons_to_sequences(
-                obj["polygon"]["path"], height=target["height"], width=target["width"],
+                obj["polygon"]["path"],
```

Comment:
If "polygon" is None, this will explode right?
Do you think we could do some validation before? Would it be a silly idea?

Reply:
idk, I didn't touch that code